server/group.go

```diff
@@ -58,6 +61,41 @@ func (a *applicationHandler) CreateGroup(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	if util.IsStringEmpty(string(newGroup.Config.Strategy.Type)) {
+		_ = render.Render(w, r, newErrorResponse("please provide a valid strategy type", http.StatusBadRequest))
+		return
+	}
```

Comment:
is this necessary since we're still going to compare to config.DefaultStrategyProvider?